src/main/java/com/google/alpollo/Song.java

```diff
@@ -1,17 +1,32 @@
 package com.google.alpollo;
 
+import com.googlecode.objectify.annotation.Index;
+
 /** Represents the structure of the song. */
 public class Song {
-  private final String artist;
-  private final String name;
-  private final String album;
+  @Index
+  private String artist;
+  @Index
+  private String name;
+  @Index
+  private String album;
+
+  public Song() { }
```

Comment:
You can make it `protected` since developers shouldn't use it. Not `private`, because `SongInfo` needs it.